src/state/quick-bar-mixin.ts

```diff
@@ -13,11 +13,14 @@ declare global {
 
 export default <T extends Constructor<HassElement>>(superClass: T) =>
   class extends superClass {
+    private isMacOS = false;
+
     protected firstUpdated(changedProps: PropertyValues) {
       super.firstUpdated(changedProps);
+      this.isMacOS = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
```

Comment:
You can take this check outside of the class:

`const isMacOs = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);`

Reply:
Done